### ISSUE TITLE
I have some questions about em1 and em2.

### ISSUE THREAD

**Issue by neulky** [daniilidis-group/event_feature_tracking#5]
My first confuse is:
This system defines two templates 'ef.prev_shifted_weights' and 'ef.template_decimated'?  Can em1 use 'ef.template_decimated' as a template?

My second confuse is:
In the em2_affine.m,Line 230:
     time_shifted_points = bsxfun(@minus, events(1:2, :), ...
          feature_pos + flow * dt) + ...
          bsxfun(@times, flow, (events(3, end) - events(3, :)));
In order to calculate 'time_shifted_points', Can this use the new feature point position and the new optical flow after the affine warping?

**Comment by alexzzhu** [comment]
ef.templated_decimated is a boolean to indicated whether the template has undergone sphere decimation. Do you mean ef.prev_shifted_points? In that case, there is the option to use this as the template for em1 to simplify the flow estimation step.

I'm not sure what you mean by the second question, but the transformation is defined as x' = A*(x-xdot * dt), so the points are time shifted, and then the affine transformation is applied.

**Comment by neulky** [comment]
Thank you Alex, 
I'm Sorry, I didn't express my questions clearly. 
In my first question, I mean that this system defines two templates 'ef.prev_shifted_points' and 'ef.template_points'. Now I understand using the template 'ef.prev_shifted_points' instead of 'ef.template_points' for em1 is simplifying the flow estimation step.
In my second question, In the em2_affine.m,Line 230:
time_shifted_points = bsxfun(@minus, events(1:2, :), feature_pos + flow * dt) + bsxfun(@times, flow, (events(3, end) - events(3, :)));
Can replace the above codes  as following:
time_shifted_points = bsxfun(@minus, events(1:2, :), **feature_pos - pos_shift** + flow * dt) + bsxfun(@times, flow, (events(3, end) - events(3, :)));
The change is replace feature_pos to feature_pos - pos_shift.

**Comment by neulky** [comment]
Yes, I replace feature_pos with feature_pos_out, the tracking still works.But by observing the results, I don't know the tracking accuracy is improving or not.
